src/Evolution/Systems/GrMhd/ValenciaDivClean/Characteristics.hpp

```diff
@@ -27,7 +27,7 @@ struct Normalized;
 
 namespace grmhd {
 namespace ValenciaDivClean {
-
+namespace approx {
```

Comment:
I don't like this namespace. In the unit testing we have an object called `approx` that is used to approximately compare two doubles. Why even bother adding a namespace to this? Especially for a simple function, which you could just rename. And the new function is in a detail namespace and follows a different naming convention than `approx::characteristic_speeds`. My suggestion is to have one commit that renames this to `characteristic_speeds_approximate_mhd`, then add the new function not in a detail namespace and keep the name `characteristic_speeds_hydro`. Moving this into the namespace actually broke the `Tags::CharacteristicSpeedsCompute` (which we don't use and doesn't appear to be tested for functionality).